tests/api2/test_cronjob.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+from time import sleep
+
+from middlewared.test.integration.utils import call, ssh
+
+TESTFILE = '/tmp/.testFileCreatedViaCronjob'
+
+
+@pytest.fixture(scope='module')
+def cronjob_dict():
+    return {}
+
+
+def creating_new_cron_job_which_will_run_every_minute(cronjob_dict):
+    results = call("cronjob.create", {
+        'user': 'root',
+        'command': f'touch "{TESTFILE}"',
+        'schedule': {'minute': '*/1'}
+    })
+    cronjob_dict.update(results)
+
+
+def checking_to_see_if_cronjob_was_created_and_enabled(cronjob_dict):
```

Comment:
None of these functions will run because they aren't named properly (i.e. pytest dictates test functions need to be named `test_*`)